pwiz_tools/Skyline/SkylineTester/TabNightly.cs

```diff
@@ -722,5 +733,51 @@ public bool UseRunningLogFile
         }
 
         BackgroundWorker SkylineTesterWindow.IMemoryGraphContainer.UpdateWorker { get; set; }
+
+        [ServiceBehavior(InstanceContextMode = InstanceContextMode.Single)]

```

Comment:
Some comments here would be good - "this interprocess communication makes it possible for a hung test run to be kept alive longer than normal so somebody can attach a debugger to it if they want" or somesuch.